javascript-bank/bank.js

```diff
@@ -0,0 +1,37 @@
+/* exported Bank */
+function Bank() {
+  this.accounts = [];
+  this.nextAccountNumber = 1;
+}
+
+Bank.prototype.openAccount = function (holder, balance) {
+  if ((balance > 0) && (balance % 1 === 0)) {
+    let acc = new Account(balance, holder);
```

Comment:
The first parameter of the `Account` constructor is the account number, not the balance.

---

![image](https://user-images.githubusercontent.com/16143771/123365770-caf07b80-d52b-11eb-8af9-e1f9be98b333.png)